src/pybind/mgr/dashboard/controllers/_base_controller.py

```diff
@@ -239,8 +239,7 @@ def _request_wrapper(func, method, json_response, xml,  # pylint: disable=unused
         def inner(*args, **kwargs):
             client_version = None
             for key, value in kwargs.items():
-                if isinstance(value, str):
-                    kwargs[key] = unquote(value)
+                kwargs[key] = unquote(value) if isinstance(value, str) else value
```

Comment:
Is this reducing the cyclomatic complexity? Theoretically, this is the exact same in cyclomatic complexity terms, but using ternary operators might trick mypy.

Reply:
Yup, I had the same feeling when I saw this. Perhaps it's reducing the number of lines (but AFAICR McCabe's doesn't account for the LoC), and hence Pylint score is higher.

Reply:
Hmm, I see what you mean! I thought it did improve the complexity because it updated value of one variable which could be similar to making a new function like this:
```python3
def unquote_value(value):
    return unquote(value) if isinstance(value, str) else value

# then calling this like: 
kwargs[key] = unquote_value(value)
```
But yes, theoretically, using ternary operators should not change cyclomatic complexity because the if-else conditions still adds a new path in the control flow graph for this method. I guess flake8 has been tricked somehow and it's not an improved complexity!
I'll look for more meaningful ways to improve the score, thanks for pointing that out.

Reply:
@VallariAg Sorry I didn't notice you left a comment, are there any updates on this? Anyways, when make check passes I'm ok approving